### ISSUE TITLE
PermissionError: [Errno 13] Permission denied

### ISSUE THREAD

**Issue by manuelregidor** [Tecnativa/doodba-copier-template#164]
Hi @Yajo ,

I'm using dooba on odoo 13 and I'm getting an error when doing docker-compose up. The user is in the docker permission group. I have done the build and download addons steps. I have given permissions to the addons folder where soft-links are created with "chmod -R ug+rwX odoo/auto". I have included the -i command to create the initial db if it does not exist and I have deleted the .empty files. I'm using ubuntu 18.04. Do you know what could be wrong?

Thank you

PostgreSQL Database directory appears to contain a database; Skipping initialization

```
db_1          | 
db_1          | 2020-08-14 07:55:26.352 GMT [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
db_1          | 2020-08-14 07:55:26.352 GMT [1] LOG:  listening on IPv6 address "::", port 5432
db_proxy_1    | INFO:root:Executing: socat tcp-listen:5432,fork,reuseaddr tcp-connect:db:5432
odoo_1        | doodba INFO: Waiting until postgres is listening at db...
odoo_1        | doodba INFO: Linking all addons from /opt/odoo/custom/src/addons.yaml in /opt/odoo/auto/addons
odoo_1        | Traceback (most recent call last):
odoo_1        |   File "/opt/odoo/common/entrypoint.d/40-addons-link", line 17, in <module>
db_1          | 2020-08-14 07:55:26.354 GMT [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
odoo_1        |     os.symlink(src, dst)
odoo_1        | PermissionError: [Errno 13] Permission denied: '../../custom/src/account_financial_reporting/account_financial_report' -> '/opt/odoo/auto/addons/account_financial_report'
odoo_1        | Traceback (most recent call last):
odoo_1        |   File "/opt/odoo/common/entrypoint", line 32, in <module>
odoo_1        |     subprocess.check_call(command)
odoo_1        |   File "/usr/local/lib/python3.6/subprocess.py", line 311, in check_call
db_1          | 2020-08-14 07:55:26.366 GMT [20] LOG:  database system was shut down at 2020-08-14 07:52:02 GMT
odoo_1        |     raise CalledProcessError(retcode, cmd)
odoo_1        | subprocess.CalledProcessError: Command '/opt/odoo/common/entrypoint.d/40-addons-link' returned non-zero exit status 1.
db_1          | 2020-08-14 07:55:26.370 GMT [1] LOG:  database system is ready to accept connections
odoo_1 exited with code 1
```

**Comment by Yajo** [comment]
Now that we have invoke, one possibility is to create the empty folders if missing during `invoke develop`. This will make us not need those `.empty` files anymore. Do you agree @joao-p-marques?

**Comment by joao-p-marques** [comment]
Yes, seems like a good option.

@estraka When a directory is empty, it is ignored by Git. Adding a "dummy" `.empty` file allows it to be discoverable by VCS. 

That's why maybe we can adopt a different strategy with dynamically generated folders with invoke.

**Comment by Yajo** [comment]
The fix path is clear then. However, this seems to be affecting some OS that is not officially supported by us (see #163), so... well, PRs welcome :blush:

**Comment by estraka** [comment]
@Yajo I will try to collaborate, but I am new to Github, Python, Odoo ... and it is costing me a lot, but I have to say that in Mac Os X 10.15, if doodba-copier-template and doodba are working for me, in case I need something install nodejs, but I did not have a clear message that this was necessary and it wasted time, in case you want to include it in "Install the dependencies"

**Comment by Yajo** [comment]
Yep, I'm sorry to hear you lost your time and I understand your frustration.

Our team already added the "fix" in our docs: #163. We're possibly not gonna make any deeper fixes, so I'm gonna close this as there's no actionable work left to do.

Thanks and keep on the good work! It's hard but joyful :blush:

**Comment by rpsjr** [comment]
Just delete odoo/custom/entrypoint.d/.empty  solved my deployment on mac os 14.0

**Comment by arielbarreiros96** [comment]
> Just delete odoo/custom/entrypoint.d/.empty solved my deployment on mac os 14.0

I know this us a closed thread but this comment was helpful to me, even though, after deleting the .empty I had to deal with the .DS_Store, which I removed with this find . -name '.DS_Store' -delete (I did it on my current folder . but this can be done with the rest of the directories). After that, rebuild the whole thing and it works
